exporter/opentelemetry-exporter-jaeger-proto/src/opentelemetry/exporter/jaeger/proto/__init__.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2018, OpenCensus Authors
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+
+OpenTelemetry Jaeger Protobuf Exporter
+--------------------------------------
+
+The **OpenTelemetry Jaeger Protobuf Exporter** allows to export `OpenTelemetry`_ traces to `Jaeger`_.
+This exporter always sends traces to the configured agent using Protobuf via gRPC.
+
+Usage
+-----
+
+.. code:: python
+
+    from opentelemetry import trace
+    from opentelemetry.exporter.jaeger.proto import JaegerExporter
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+    trace.set_tracer_provider(TracerProvider())
+    tracer = trace.get_tracer(__name__)
+
+    # create a JaegerExporter
+    jaeger_exporter = JaegerExporter(
+        service_name='my-helloworld-service',
+        # optional: configure collector
+        # collector_endpoint='http://localhost:14268/api/traces?format=jaeger.thrift',
```

Comment:
thrift? I think the default collector endpoint for grpc is `localhost:14250` and it shouldn't include the scheme?
```suggestion
        # collector_endpoint='localhost:14250',
```